packages/jest-mock-apollo/README.md

```diff
@@ -13,16 +13,32 @@ $ yarn add @shopify/jest-mock-apollo
 
 ## Setup
 
-This package provides a method, `registerSchema`, which should be called in the jest `setup` file. For example:
+This package provides a decorator for building mock GraphQL clients specific to your project. The initial function call takes in project-wide configuration and emits a customized decorator to be used across your project. For example:
 
 ```ts
-registerSchema({
-  schemaBuildPath: './fixtures/schema.graphql',
-  unionOrIntersectionTypes,
-});
+// tests/utilities/graphql.ts
+import {buildSchema} from 'graphql';
+import configureClient from '@shopify/jest-mock-apollo';
+
+// You likely want to import this from somewhere in real production code
+const schemaSrc = `
+  type Product {
+    id: Int!
+    title: String
+  }
+
+  type Query {
+    products: [Product]
+  }
+`;
+const schema = buildSchema(schemaSrc);
+
+const createGraphQLClient = configureClient({schema});
```

Comment:
`configureClient` is a bit of a weird name 😬, not sure what to call this without being super verbose though.